src/sagemaker/jumpstart/types.py

```diff
@@ -2154,3 +2154,92 @@ def __init__(
         self.data_input_configuration = data_input_configuration
         self.skip_model_validation = skip_model_validation
         self.source_uri = source_uri
+
+
+class BaseDeploymentConfigDataHolder(JumpStartDataHolderType):
```

Comment:
Will this be used for other servers too?

Reply:
Can be.